src/wrapper.js

```diff
@@ -4,26 +4,18 @@ import Autocomplete from './autocomplete'
 function accessibleAutocomplete (opts) {
   if (!opts.element) { throw new Error('element is not defined') }
   if (!opts.id) { throw new Error('id is not defined') }
-  if (!opts.source) { throw new Error('source is not defined') }
+  if (!opts.source && !opts.values) { throw new Error('values or source are not defined') }
```

Comment:
This needs a README change; add a header and short description for this new [Required option](https://github.com/alphagov/accessible-autocomplete#required-options).

Also update the `source` description so it's clear that you can specify one or the other.